ee/tabby-webserver/src/service/mod.rs

```diff
@@ -104,8 +107,14 @@ impl ServerContext {
         }
     }
 
+    /// Returns whether a request is authorized to access the content, and the user ID if authentication was used.
     async fn authorize_request(&self, request: &Request<Body>) -> (bool, Option<ID>) {
         let path = request.uri().path();
+        if demo_mode()
+            && (path.starts_with("/v1/completions") || path.starts_with("/v1beta/chat/completions"))
```

Comment:
any reason we cannot ban `/v1` `/v1beta` entirely?

Reply:
The rest of the endpoints are needed, to see all of the configuration. If all of /v1 and /v1beta are disabled, the homepage doesn't load, nor the cluster information page.